Andriy_Povzun/HW7_Andriy_Povzun/Task2.py

```diff
@@ -0,0 +1,9 @@
+import math
+
+
+def calculate_square(side_square: int):
+    perimeter_square = side_square * 4
+    diagonal_square = side_square * math.sqrt(2).__round__(2)
+    area_square = math.sqrt(side_square)
+    tuple_results = (perimeter_square, diagonal_square.__round__(2), area_square.__round__(2))
+    return tuple_results
```

Comment:
you can just code: 
return perimeter_square, diagonal_square.__round__(2), area_square.__round__(2)

and it will be the same as:
tuple_results = (perimeter_square, diagonal_square.__round__(2), area_square.__round__(2))
    return tuple_results

Reply:
got it, okay